translations/translations.json

```diff
@@ -6126,7 +6126,9 @@
       "shelterType": "Art des Wohnraums",
       "startDate": "frühestens verfügbar",
       "availableBeds": "verfügbare Betten",
-      "duration": "mögliche Aufenthaltsdauer"
+      "duration": "mögliche Aufenthaltsdauer",
+      "facetBeds": "min. Anzahl Betten",
```

Comment:
What is a facet? :see_no_evil: I think `Anzahl Betten` is enough, you also do that for english already.

Reply:
facet is the regular name for a filter

Reply:
Never heard that name, but fine. I'd just have called it filter.

Reply:
you can also rename it when we have checked ukrainian translations if you prefer 'filter'. i'm fine with that

Reply:
no its fine, just had never heard that. but has no benefit so lets just keep it